packages/realm/bindgen/src/templates/node-wrapper.ts

```diff
@@ -46,6 +46,9 @@ export function generate({ rawSpec, spec: boundSpec, file }: TemplateContext): v
     // TODO: Remove the need to store Realm as a global
     // @see https://github.com/realm/realm-js/issues/2126
     const nativeModule = global.__RealmFuncs;
+    if(!nativeModule) {
+      throw new Error("Could not find the Realm binary. Please consult our troubleshooting guide. (https://www.mongodb.com/docs/realm-sdks/js/latest/#md:troubleshooting-missing-binary)");
```

Comment:
I'm not sure, but perhaps some terminals would have an easier time constructing a link if it's not wrapped in parentheses. Just a thought.

```suggestion
      throw new Error("Could not find the Realm binary. Please consult our troubleshooting guide: https://www.mongodb.com/docs/realm-sdks/js/latest/#md:troubleshooting-missing-binary");
```

Reply:
When I tested it, it worked fine, but maybe a good idea if people need to actually manually copy and paste it out.